jdaviz/configs/default/plugins/export_plot/export_plot.vue

```diff
@@ -95,9 +95,25 @@
                 </v-col>
               </v-row>
               <v-row v-if="movie_msg===''" justify='end'>
+                <v-tooltip top v-if="movie_recording">

```

Comment:
I am not sure about moving this. When I click save movie, the stop button pops up over the same mouse location, so it is slightly faster if I want to stop recording.

Reply:
> the stop button pops up over the same mouse location

Only if you were clicking on the very right of the button.

But maybe we should just have the button swap text entirely rather than adding a new button and disabling the original.  I just think now that its aligned right (like all other buttons in plugins), the sudden shift to the right introduced by another button is jarring.

Reply:
OK, leave it as-is then. I'd rather not have to figure out the logic of the same button doing different thing right now. (Though if you want to, might be similar to play/pause in Slice.)